include/h2o/socket/uv-binding.h

```diff
@@ -62,6 +62,11 @@ static inline uint64_t h2o_now(h2o_loop_t *loop)
     return uv_now(loop);
 }
 
+static inline uint64_t h2o_now_nanosec(h2o_evloop_t *loop)
+{
+    return uv_hrtime();
```

Comment:
`uv_hrtime` is not based on the unix epoch. I think we need to either use something other or simply do `uv_now() * 1000000`.

Reply:
Thank you. I misunderstood the API. I also noticed that the signature of this function is mildly wrong. I believe `h2o_evloop_t` should be `h2o_loop_t`. Will fix together.